pages/mesosphere/dcos/services/elastic/2.7.0-6.8.1/how-to-guides/exposing-kibana/index.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: layout.pug
+navigationTitle: Kibana and Edge-LB    
+excerpt: Exposing Kibana using Edge-LB
+title: Kibana and Edge-LB
+menuWeight: 20
+model: /mesosphere/dcos/services/elastic/data.yml
+render: mustache
+---
+
+# Expose Kibana using Edge-LB
+
+## Install Edge-LB
```

Comment:
I think for this page, this is the least confusing approach: three main headings and smaller sub-headings so they show up on the right hand TOC.